content/influxdb/cloud-dedicated/admin/databases/_index.md

```diff
@@ -89,22 +100,33 @@ operating cost of your cluster.
 
 ### Column limit
 
-**Default maximum number of columns**: 250
+**Default maximum number of columns**: 1000
```

Comment:
I believe the default remains at 250 - [this work](https://github.com/influxdata/influxdb_iox/issues/11156) was done to prevent users further raising their default column limit to the point it became problematic for the querier.

My understanding is:

* By default, a column limit of 250 is imposed per table (the "soft limit").
* This limit can be raised (via a gRPC call, I'm not sure by who - support?) to up to 1000.
* Existing tables that had >1000 columns at the time the hard limit was introduced continue to work, but their column limit cannot be raised further.

Really we do not want users hitting 1000 columns, but we needed to keep some flexibility - I believe performance starts to degrade above the default 250 soft limit (or at least that is why it was chosen at the time).

Reply:
I believe some customers would trade performance for width and perhaps others would not.  Ideally, they would like to know more about the tradeoff ... ie how much performance they will trade as they push the limit and I doubt we can quantify it for them.  But, perhaps there is a middle ground in which we give the customer the ability to move the limit but warn them of a likely decay in performance and recommend that they watch for the down-stream impact.  Are there any metrics in the Observability Dashboard that we could point to as an indicator that they might be widening their table too much?

Reply:
It's not a grandual-perf-trade-off thing above ~1,000 columns; the queriers and (especially) the catalog service begin to fail outright.

Engineering took the decision to cap the columns after a number of serious and time-consuming incidents in caused by multiple customers just yolo'ing the limit as high as they wanted :(

Reply:
Got it, thanks.
So what is the verdict?  I was on a call yesterday with VAST and got their hopes up regarding 1000 columns.  Shall I return to them with the news that the limit remains 250?  Is there a longer view on this that I can share, possibly an effort to re-architecture or move a bottleneck somewhere in the data pipeline?

Reply:
The default limit is 250, but it can be raised to 1000 so they should be 👌 

We prevent the limit from being raised above 1,000 though :+1:

I'm not sure if it is an area being actively worked on or planned for.

Reply:
> I believe the default remains at 250 - [this work](https://github.com/influxdata/influxdb_iox/issues/11156) was done to prevent users further raising their default column limit to the point it became problematic for the querier.

@domodwyer ❤️ Thanks for the clarification!